library/alloc/src/vec.rs

```diff
@@ -296,6 +296,9 @@ use crate::raw_vec::RawVec;
 /// [owned slice]: Box
 /// [slice]: ../../std/primitive.slice.html
 /// [`&`]: ../../std/primitive.reference.html
+#[doc(alias = "ArrayList")]
+#[doc(alias = "List")]
+#[doc(alias = "Vector")]
```

Comment:
```suggestion
#[doc(alias = "array_list")]
#[doc(alias = "list")]
#[doc(alias = "vector")]
```
We should keep it `snake_case` like rust style, python also called it `list`, not `List` (although that is the typing type).

Reply:
Oh, I forgot this is a struct, can just ignore it. Sorry.